src/components/metadata/MetaSimple.js

```diff
@@ -83,36 +70,7 @@ export class MetaSimple extends Component {
           value={fieldValue}
           ref="imagepicker"
         />
-        <span className="images-wrapper">
-          <button onClick={this.handleOpenModal}>
-            <Icon name="picture-o" />
-          </button>
-          <Modal
-            isOpen={this.state.showModal}
-            onAfterOpen={this.fetchStaticFiles}
```

Comment:
Not fetching static files now?

Reply:
As noted in https://github.com/jekyll/jekyll-admin/pull/560#discussion_r357512404, this is a development artifact.
To clarify, the child element of the Modal, `StaticIndex` handles the fetch..